### DISCUSSION TITLE
stat_jobの変更

### DISCUSSION THREAD

**Discussion by nim-hrkn** [Tomoki-YAMASHITA/CrySPY#28] [General]
構造最適化プログラムをLAMMPSとします。
cryspy.py一回目で各構造のstat_jobがsubmittedになります。
self.job_stat[cid] = 'done'に変えているところはcheck_job. 
check_jobはstat_jobの３行目を読む。
構造最適化プログラムが終了した後でstat_jobをsubmittedからdoneに変えているのはどこでしょうか。

**Comment by nim-hrkn** [comment]
Pyhonソースを見ても発見できなくて、job_cryspyで行っていた。
sed -i -e '3 s/^.*$/done/' stat_job

**Comment by Tomoki-YAMASHITA** [comment]
そうですね．そこで書き換えてます．
現状，ジョブの最後で終了判定（submitted --> doneに書き換え）をするようにしています
